javascript/javascript3/week3/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>canvas and circles</title>
+</head>
+<body>
+    <canvas id="myCanvas"></canvas>
+
+
+    
+<script src="Lets-make-some-art-using-classes.js"></script>
+</body>
+</html>
```

Comment:
I think you have pushed some wrong files into this repo, also rename this HTML file so that it shows it belongs to the circle art! 😄

Reply:
you mean .. it shows all javascript3 homeworks is that or somethings else.if not can you please explain what is it ...   actually i merged to master until week2 .
week3 only left for review... i will do changes